iota/transaction/creation.py

```diff
@@ -488,3 +488,65 @@ def _create_input_transactions(self, addy):
                 # Note zero value; this is a meta transaction.
                 value=0,
             ))
+
+    def add_signature_or_message(
+            self,
+            fragments,  # type: Iterable[Fragment]
+            start_index=0  # type: Optional(int)
+    ):
+        # type: (Iterable[Fragment], int) -> None
```

Comment:
```suggestion
        # type: (...) -> None
```

If the individual parameters are type-hinted, it is not necessary to repeat them in the function type hint.
https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code